README.md

```diff
@@ -117,8 +117,8 @@ The following template variables need to be replaced as follows with values from
 * *\<vCenter Server FQDN/IP\>*: IP address or FQDN of the vCenter server
 
 *IMPORTANT*:
-- The template used in the MachineSet must be a Windows Server 1909
-  image as described in [vSphere prerequisites](docs/vsphere-prerequisites.md).
+- The template used in the MachineSet must be an image with a supported Windows Server
+  version, as described in [vSphere prerequisites](docs/vsphere-prerequisites.md).
```

Comment:
This wording seems off. How about something like this?

`The VM template provided in the MachineSet must use a supported Windows Server version...`

Basing this off [this doc](https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-F40130B0-0194-4A41-91FA-1A967721924B.html), in that a template is a master copy of a VM, so the `image` language used seems a bit redundant.